tools/azsdk-cli/Azure.Sdk.Tools.Cli/Prompts/Templates/ReadMeGenerationTemplate.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Azure.Sdk.Tools.Cli.Prompts.Templates;
```

Comment:
I'd move this to be in the ReadmeTool's folder, not here, since it's really an implementation detail and not used by others.

Reply:
I wanted to store all the templates in one central location.